contracts/MarginCalculator.sol

```diff
@@ -281,12 +299,17 @@ contract MarginCalculator is Ownable {
         uint256 _collateralDecimals,
         bool _isPut
     ) external view returns (uint256) {
+        // get product hash
         bytes32 productHash = _getProductHash(_underlying, _strike, _collateral, _isPut);
 
+        // scale short amount from 1e8 to 1e27
         FPI.FixedPointInt memory shortAmount = FPI.fromScaledUint(_shortAmount, BASE);
+        // scale short strike from 1e8 to 1e27
         FPI.FixedPointInt memory shortStrike = FPI.fromScaledUint(_strikePrice, BASE);
```

Comment:
These comments are not helpful. Either rename the `fromScaledUint` function name to make it self explaining or remove the comments.

Reply:
why do u think it is not helpful ?